static/app/utils/analytics/profilingAnalyticsEvents.tsx

```diff
@@ -2,6 +2,12 @@ export type ProfilingEventParameters = {
   'profiling_views.go_to_flamegraph': {source: string};
   'profiling_views.landing': {};
   'profiling_views.onboarding': {};
+  'profiling_views.onboarding_action.dismiss': {
+    action: 'dismiss';
+  };
+  'profiling_views.onboarding_action.done': {
+    action: 'done';
+  };
```

Comment:
thoughts on sending this under the same analytic event like `profiling_views.onboarding_action` and just use `action` to see what was clicked instead of 2 separate events?

Reply:
less code, works for me :D